merkletree/pad.go

```diff
@@ -149,10 +146,18 @@ func (pad *PAD) Sign(msg ...[]byte) []byte {
 	return pad.signKey.Sign(bytes.Join(msg, nil))
 }
 
+func (pad *PAD) VrfPublicKey() vrf.PublicKey {
```

Comment:
I'm not sure if this method would be used anywhere other than `directory.SetPolicies`. Why couldn't we just use `directory.policies.VrfPublicKey()` in `directory.SetPolicies` instead? This seems to be a workaround when we don't have VRF key rotation yet, right?